docs/k0s-in-docker.md

```diff
@@ -23,6 +25,12 @@ You can run your own k0s in Docker:
 docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.io/k0sproject/k0s:latest
 ```
 
+**Note:** If you are using Docker Desktop as the runtime, starting from 4.3.0 version it's using cgroups v2 in the VM that runs the engine. This means you have to add some extra flags to the above command to get kubelet and containerd to properly work with cgroups v2:
+
+```sh
+--cgroupns=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw
```

Comment:
Just out of curiosity:

I reckon those are the changes made in weaveworks/footloose#272. Wondering what the cgroupns flag is really doing. From my Linux box's manpage:

>       --cgroupns=""
>           Set the cgroup namespace mode for the container.
>             host:    run the container in the host's cgroup namespace
>             private: run the container in its own private cgroup namespace
>             "":      (unset) use the daemon's default configuration (host on cgroup v1, private on cgroup v2)

What is the difference here when using the private cgroups namespace?